sdk/loadtestservice/Azure.Developer.LoadTesting/samples/Sample1_HelloWorld.md

```diff
@@ -0,0 +1,15 @@
+# <scenario_list>
+
+To use these samples, you'll first need to set up resources. See [getting started](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/loadtestservice/Azure.Developer.LoadTesting/README.md#getting-started) for details.
+
+## <scenario>
+
+You can create a client and call the client's `<operation>` method
+
+```C# Snippet:Azure_Developer_LoadTesting_Scenario
```

Comment:
remove for now, until we have actual sample code

Reply:
Added samples here